governance.md

```diff
@@ -153,49 +289,135 @@ are enumerated below.
 
 ## Other duty-specific github roles:
 - [**Github Org Owner**](https://github.com/orgs/kubernetes/people?utf8=%E2%9C%93&query=%20role%3Aowner):
-  can create repos, do ~any github action; the number of
+  can create repos, and do any github action; the number of
```

Comment:
I agree we should try to remove google-admin and googlebot, but not in the middle of the release crunch.